src/vulkan/vk-device.cpp

```
@@ -30,6 +30,11 @@
 
 namespace rhi::vk {
 
+static constexpr VkSubgroupFeatureFlags kWaveOpsSubgroupFeatureMask =
+    VK_SUBGROUP_FEATURE_BASIC_BIT | VK_SUBGROUP_FEATURE_VOTE_BIT | VK_SUBGROUP_FEATURE_ARITHMETIC_BIT |
```

Cppcheck (2.20.0)

[style] The function 'releasePages' is never used.

(unusedFunction)